src/index.ts

```diff
@@ -72,7 +80,15 @@ const plugin = (moduleName: string = "external", sourceMaps = true): Plugin => {
 
                   if (req && req.value.includes("node_modules")) {
                     const { start, end } = req;
-                    const newPath = req.value.replace(/node_modules/g, moduleName);
+                    // compute a new path relative to the bundle root
+                    const bundlePath = replace(
+                      path.join(path.dirname(fileName), req.value)
+                    );
+                    // and then format the path relative the updated chunk path
+                    const newPath = path.relative(
```

Comment:
#6 this is returning path without ./ if file is in root directory

Reply:
yep I forgot about this behavior. The returned path needs processed with something like `newPath.startsWith('.') ? newPath : './' + newPath`